### PR TITLE
MuhammedKalkan.OpenLens version 6.5.2.286

### PR DESCRIPTION
**Pull request by brotich**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108932)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/m/MuhammedKalkan/OpenLens/6.5.2.286/MuhammedKalkan.OpenLens.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+PackageIdentifier: MuhammedKalkan.OpenLens
+PackageVersion: 6.5.2.286
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/m/MuhammedKalkan/OpenLens/6.5.2.286/MuhammedKalkan.OpenLens.installer.yaml:13]:
```suggestion
- Architecture: x64
```

File: manifests/m/MuhammedKalkan/OpenLens/6.5.2.286/MuhammedKalkan.OpenLens.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://github.com/MuhammedKalkan/OpenLens/releases/download/v6.5.2.286/OpenLens-6.5.2.286.exe
+  InstallerSha256: 15E5FCF5A8F22B78E25AC2714894CB367F91222E82CB6BEC9E96FBED96D76FEB
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/m/MuhammedKalkan/OpenLens/6.5.2.286/MuhammedKalkan.OpenLens.installer.yaml:20]:
```suggestion
- Architecture: x64
```

File: manifests/m/MuhammedKalkan/OpenLens/6.5.2.286/MuhammedKalkan.OpenLens.installer.yaml (hunk 1 above)
Comment (brotich) [reply manifests/m/MuhammedKalkan/OpenLens/6.5.2.286/MuhammedKalkan.OpenLens.installer.yaml:13]: Thanks. I dont know i forgot about this

Comment (brotich) [inline manifests/m/MuhammedKalkan/OpenLens/6.5.2.286/MuhammedKalkan.OpenLens.installer.yaml:13]:
```suggestion
- Architecture: x64
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**